# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/DMABufLibWrapper.cpp
# Commit: 8f51b53b700c
# Full Hash: 8f51b53b700caee0f5976a74bcd41762a54b623c
# Author: stransky <stransky@redhat.com>
# Date: 2025-02-18 04:51:41
# Description:
#   Bug 1946985 [Linux] Get RGB alpha dmabuf format if Wayland compositor doesn't provide modifiers for it r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D238517
# ==============================================================================

diff -r c7e24e2d9124 -r 8f51b53b700c widget/gtk/DMABufLibWrapper.cpp
--- a/widget/gtk/DMABufLibWrapper.cpp	Mon Feb 17 20:10:07 2025 +0000
+++ b/widget/gtk/DMABufLibWrapper.cpp	Mon Feb 17 20:14:03 2025 +0000
@@ -267,7 +267,7 @@
 void DMABufDevice::GetModifiersFromGfxVars() {
   mFormatRGBX =
       new DRMFormat(GBM_FORMAT_XRGB8888, gfxVars::DMABufModifiersXRGB());
-  mFormatRGBX =
+  mFormatRGBA =
       new DRMFormat(GBM_FORMAT_ARGB8888, gfxVars::DMABufModifiersARGB());
 }
 
